test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -671,6 +699,78 @@ describe('33acrossBidAdapter:', function () {
         const [ buildRequest ] = spec.buildRequests(bidRequests);
         validateBuiltServerRequest(buildRequest, serverRequest);
       });
+
+      it('returns the viewport dimensions of the top most accessible frame', function() {
+        const ttxRequest = new TtxRequestBuilder()
+          .withBanner()
+          .withDevice({
+            ext: {
+              ttx: {
+                viewport: {
+                  w: 9876,
+                  h: 5432
+                }
+              }
+            }
+          })
+          .withProduct()
+          .build();
+        const serverRequest = new ServerRequestBuilder()
+          .withData(ttxRequest)
+          .build();
+        const notAccessibleParentWindow = {};
+
+        Object.defineProperty(notAccessibleParentWindow, 'document', {
+          get() { throw new Error('fakeError'); }
+        });
+
+        sandbox.stub(win, 'parent').value({
+          document: {
+            documentElement: {
+              clientWidth: 1234,
+              clientHeight: 4567
+            }
+          },
+          parent: {
+            parent: notAccessibleParentWindow,
+            document: {
+              documentElement: {
+                clientWidth: 9876,
+                clientHeight: 5432
+              }
+            },
+          }
+        });
+
+        const [ buildRequest ] = spec.buildRequests(bidRequests);
+        validateBuiltServerRequest(buildRequest, serverRequest);
+      });
+    });
+
+    describe('when the window height is greater than the width', function() {
+      it('returns the smaller dimension as the width', function() {
```

Comment:
What about testing landscape mode?

Reply:
Technically, the landscape scenario is covered in all the other unit tests, because they all use `validateBuiltServerRequest` for the assertion, and the default screen dimensions for the window mock are landscape.
Either way I've decided to add a spec for the landscape scenario just to make it more clear that we are testing that specific scenario.